coordinator/src/app_desc/genesis.rs

```diff
@@ -2,34 +2,34 @@
 // This file is part of CodeChain.
 //
 // This program is free software: you can redistribute it and/or modify
-// it under the terms of the GNU General Public License as
+// it under the terms of the GNU Affero General Public License as
 // published by the Free Software Foundation, either version 3 of the
 // License, or (at your option) any later version.
 //
 // This program is distributed in the hope that it will be useful,
 // but WITHOUT ANY WARRANTY; without even the implied warranty of
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU General Public License for more details.
+// GNU Affero General Public License for more details.
 //
-// You should have received a copy of the GNU General Public License
+// You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-use super::Seal;
-use crate::bytes::Bytes;
-use crate::hash::H256;
-use crate::uint::Uint;
+use super::bytes::Bytes;
+use super::hash::H256;
+use super::seal::Seal;
 use ckey::PlatformAddress;
+use serde::Deserialize;
 
 /// Scheme genesis.
-#[derive(Debug, PartialEq, Deserialize)]
+#[derive(Debug, PartialEq, Deserialize, Default)]
 #[serde(rename_all = "camelCase")]
 pub struct Genesis {
```

Comment:
Should we allow to specify this many parameters to construct a genesis block's header? This question may be my ignorance, but I think we should minimize what chain creators need to specify. Please enlighten me on this.

Reply:
I agree with you. Most of the fields are not important.
As far as I know, when we want to create a new version of the test chain, we changes a small data in genesis header to get a new genesis hash. We can easily ignore all blocks from different genesis hash.

How about remaining only the `extraData` field?

Reply:
That's more minimal than what I expected. If `extraData` is enough, then I'm ok with it.